### DISCUSSION TITLE
Share screen for the Rooms

### DISCUSSION THREAD

**Discussion by MRKSC** [benawad/dogehouse#2192] [Q&A] [answered]
It will be great if moderator is able to share his local screen and use it for better visual experience when he tries to teach or showcase something.

**Comment by dk-raw** [comment, marked as answer]
DogeHouse won't have a share screen feature cause it is audio first

**Comment by BronzW** [comment]
Ben has also said he does not want to implement this sort of feature.

**Comment by 8cells** [comment]
But is it easy to do? (also webcam sharing)
